packages/resource-deployment/templates/rest-api-templates/model-products.template.json

```diff
@@ -0,0 +1,48 @@
+    "parameters": {
+        "apimServiceName": {
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "properties": {
+                "description": "Subscribers will be able to run 5 calls/minute up to a maximum of 100 calls/week.",
+                "terms": "",
+                "subscriptionRequired": true,
+                "approvalRequired": false,
+                "subscriptionsLimit": 1,
+                "state": "published",
+                "displayName": "Starter"
+            },
+            "name": "[concat(parameters('apimServiceName'), '/starter')]",
+            "type": "Microsoft.ApiManagement/service/products",
+            "apiVersion": "2019-01-01"
+        },
+        {
+            "properties": {
+                "value": "<!--\r\n            IMPORTANT:\r\n            - Policy elements can appear only within the <inbound>, <outbound>, <backend> section elements.\r\n            - Only the <forward-request> policy element can appear within the <backend> section element.\r\n            - To apply a policy to the incoming request (before it is forwarded to the backend service), place a corresponding policy element within the <inbound> section element.\r\n            - To apply a policy to the outgoing response (before it is sent back to the caller), place a corresponding policy element within the <outbound> section element.\r\n            - To add a policy position the cursor at the desired insertion point and click on the round button associated with the policy.\r\n            - To remove a policy, delete the corresponding policy statement from the policy document.\r\n            - Position the <base> element within a section element to inherit all policies from the corresponding section element in the enclosing scope.\r\n            - Remove the <base> element to prevent inheriting policies from the corresponding section element in the enclosing scope.\r\n            - Policies are applied in the order of their appearance, from the top down.\r\n        -->\r\n<policies>\r\n\t<inbound>\r\n\t\t<rate-limit calls=\"5\" renewal-period=\"60\" />\r\n\t\t<quota calls=\"100\" renewal-period=\"604800\" />\r\n\t\t<base />\r\n\t</inbound>\r\n\t<backend>\r\n\t\t<base />\r\n\t</backend>\r\n\t<outbound>\r\n\t\t<base />\r\n\t</outbound>\r\n</policies>",
+                "format": "xml"
+            },
+            "name": "[concat(parameters('apimServiceName'), '/starter/policy')]",
+            "type": "Microsoft.ApiManagement/service/products/policies",
+            "apiVersion": "2019-01-01",
+            "dependsOn": ["[resourceId('Microsoft.ApiManagement/service/products', parameters('apimServiceName'), 'starter')]"]
+        },
+        {
+            "properties": {
+                "description": "Subscribers have completely unlimited access to the API. Administrator approval is required.",
+                "subscriptionRequired": true,
+                "approvalRequired": true,
```

Comment:
what is this approvalRequired & subscriptionRequired used for?

Reply:
subscriptionRequired is to enforce subscription to the product. We don't need it. If you have subscription required you can choose to have approval vetted which is what the other setting is for.

Reply:
Set both to false